### DISCUSSION TITLE
Use of transposition models at a sub-hourly time-step

### DISCUSSION THREAD

**Discussion by kurt-rhee** [pvlib/pvlib-python#1949] [Q&A]
Hello pvlib community, 

I was reading through these Perez papers and go to page 141/176 (C-26 of "An Anisotropic Hourly...Perez 1985") In section 2.3.3 the method for testing against 1 minute data is described.  Results are described in section 3.2.2.  

MBE and RMSE are presented in the paper for the Perez model using hourly values, I was wondering, has there been any further research to quantify the MBE and RMSE of the Perez model using non-hourly values?

[Perez_Transposition.pdf](https://github.com/pvlib/pvlib-python/files/13983033/Perez_Transposition.pdf)

**Comment by adriesse** [comment]
We reported on both 1-minute and hourly values here: [DOI: 10.1016/j.solener.2023.112093](http://dx.doi.org/10.1016/j.solener.2023.112093) and I think you'll find other reports of sub-hourly accuracy in the literature.
